pages/whattodo/basic-github-commands/index.js

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import Navbar from "../../components/Navbar";
+import Footer from "../../components/Footer";
+import GitCom from "../../components/gitCom";
```

Comment:
Make the path form `../../components/gitCom` to `../../components/GitCom` . 

The reason is not type sensitive about casing but while deployment it gets deploys in a linux machine which is type sensitive so it will not be able to find the file in the path and deployment will fail.

make these changes and make a new PR. @pratham4434